### DISCUSSION TITLE
Tailwindcss Standalone CLI 3.1.6 does not seem to work

### DISCUSSION THREAD

**Discussion by murugappanrm** [tailwindlabs/tailwindcss#8910] [Help]
**What version of Tailwind CSS are you using?**

Tailwindcss version 3.1.6

**What build tool (or framework if it abstracts the build tool) are you using?**

Standalone CLI under Tailwindcss version 3.1.6

**What version of Node.js are you using?**

Not Applicable

**What browser are you using?**

Chrome, Firefox

**What operating system are you using?**

Window 10

**Reproduction URL**

Have attached a zip file containing the code
[tailwindcss.zip](https://github.com/tailwindlabs/tailwindcss/files/9149099/tailwindcss.zip)

**Describe your issue**

I tried using the Standalone CLI as suggested. Initially when i define the page and tailwind details and generated the css style sheet using the tailwindcss.exe, it all seems to work great. The problem came when i changed any of the classes, especially color related. The page display incorrect contents in the styling. The change only gets applied after the generate the style sheet again using tailwindcss.exe.

**Comment by adamwathan** [comment]
Hey! Are you running the CLI in watch mode?

**Reply by arisac** [reply]
@adamwathan for ref: I have previously answered this (same question) on #8904
